source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+  ```
+  ```{math}
+  P(s) = C_0 + C_1 , s + C_2 , s^2 + C_3 , s^3 + , \ldots
+  ```
+  The {math}`C_0` term simply shifts where the edge of the bend is. If all the {math}`C_n` are zero except for
+  {math}`C_0` and {math}`C_1` then
+  ```{math}
+  C_1 = \frac{1}{2 \cdot \text{field_int}}
+  ```
+%
+- **g, rho (output param)**
+The Reference bending radius which determines the reference coordinate system is `rho` (see
+[](#s:ref)). `g` = `1/rho` is the "bend strength" and is proportional to the Reference
+dipole magnetic field. `g` is related to the reference magnetic field `bend_field_ref` via
+  ```{math}
+  :label: gqpb
+
+  \text{g} = \frac{q}{p_0} \cdot \text{bend_field_ref}
+end{equation}
+where {math}`q` is the charge of the reference particle and {math}`p_0` is the reference momentum. It is
+important to keep in mind that changing `g` will change the Reference orbit ([](#s:coords.3)) and
+hence will move all downstream lattice elements in space.
+
+  The total bend strength felt by a particle is the vector sum of `g` plus the zeroth order
+magnetic multipole. If the multipole `tilt0` and `Ks0` is zero, the total bend strength is
+  ```{code} yaml
+  norm_bend_field = g + Kn0
+  ```
+  Changing the multipole strength `Kn0` or `Ks0` leaves the Reference orbit and the positions of
+all downstream lattice elements
+unchanged but will vary a particle's orbit. One common mistake when designing lattices is to vary
+`g` and not `Kn0` which results in downstream elements moving around. See Sref{s:ex.chicane}
+for an example.
+
+  Note: A positive `g`, which will bend particles and the reference orbit in the {math}`-x` direction
+represents a field of opposite sign as the field due a positive `hkick`.
+%
+- **h1, h2**
+The attributes `h1` and `h2` are the curvature of the entrance and exit pole faces.
+%
+- **L, L_arc, L_chord, L_sagitta (output param)** 
+The `Length` parameter, is the arc length of the reference trajectory through the bend.
+
+  `L_chord` is the chord length from entrance point to exit point.
+The `L_sagitta` parameter is the sagitta length (The sagitta is the distance
+from the midpoint of the arc to the midpoint of the chord). `L_sagitta` can be negative and will have
+the same sign as the `g` parameter.
+%
+- **L_rectangle**
+The `L_rectangle` parameter is the "rectangular" length defined to be the distance between the
+entrance and exit points. The coordinate system used for the calculation is defined by the setting
+of `fiducial_pt`. {numref}`f:rbend` shows `l_rectangle` for `fiducial_pt` set to
+`entrance_end` (the coordinate system corresponds to the entrance coordinate system of the bend).
+In this case, and in the case where `fiducial_pt` is set to `exit_end`, the rectangular
+length will be {math}`rho sinalpha`. If `fiducial_pt` is set to `none` or `center`,
+`l_rectangle` is the same as the chord length.
+%
+- **ref_tilt**
+The `ref_tilt` attribute rotates a bend about the longitudinal axis at the entrance face of the
```

Comment:
In the list on lines 11-22, this is called `tilt_ref`.

Reply:
Changed everything to `tilt_ref`.